Update to Tychonoff corkscrew (S88) (#1464)
pi-base/data: spaces/S000088/properties/P000198.md

---
space: S000088
property: P000198
value: true
---

> The space admits a closed cover consisting of two singletons and

Reviewer: ```suggestion
$X$ is the union of two singletons and
```
("closed cover" was unnecessary.  A countable union of subspaces with the property is all that's needed.)
